### DISCUSSION TITLE
[@types/tar] Definition for Pack class is completely broken

### DISCUSSION THREAD

**Discussion by NemoStein** [DefinitelyTyped/DefinitelyTyped#63666] [Issues with a @types package]
The type definition for the low-level API class tar.Pack() does not represent the actual class.
`Pack` is a MiniPass class extension, but it is defined as a regular function:
```
export function Pack(props?: HeaderProperties): PackStream;
```
The `props` parameter is also wrong.
In the current state, it's impossible to use the low-level API

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
@NemoStein Hi, please review https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63701
